actions/auth_actions.js

```diff
@@ -229,56 +229,57 @@ export const changeServerStatusText = text => {
 /**
  * The user forgot their password and is submitting their email address to request a link
  */
+// TODO: handle status message with reducers while working on auth screen refactor
 export const sendResetPasswordRequest = email => {
-    return dispatch => {
+    return async dispatch => {
         // setting isSubmitting to true
         // shows loader on button
         dispatch({ type: SUBMIT_START });
-        return axios(URL + '/api/password/email', {
-            method: 'POST',
-            data: {
-                email: email,
-                api: true // we need this to override the response
-            },
-            headers: {
-                Accept: 'application/json',
-                'Content-Type': 'application/json'
-            }
-        })
-            .then(response => {
-                console.log('sendResetPasswordRequest', response.data);
-
-                if (
-                    response.data.message ===
-                    'We have emailed your password reset link!'
-                ) {
-                    // setting isSubmitting to false
-                    dispatch({ type: SUBMIT_END });
-                    return {
-                        success: true
-                    };
-                }
-            })
-            .catch(error => {
-                console.log('sendResetPasswordRequest', error.response.data);
-
-                if (
-                    error.response.data.message ===
-                    'The given data was invalid.'
-                ) {
-                    // setting isSubmitting to false
-                    dispatch({ type: SUBMIT_END });
-                    return {
-                        success: false,
-                        msg: error.response.data.errors.email // We can't find a user with that email address
-                    };
+        let response;
+        try {
+            response = await axios(URL + '/api/password/email', {
+                method: 'POST',
+                data: {
+                    email: email,
+                    api: true // we need this to override the response
```

Comment:
Is this still needed? Seems irrelevant and looks like an early way of fixing something quickly

Reply:
Will check this in remove if not needed.. will add it in future PR..
Made a trello card to track this
[trello card](https://trello.com/c/3sgbZfB0)